### PR TITLE
Use custom CSS files

### PR DESCRIPTION
**Pull request by vmcj**
Although a simple fix, I'm not sure if this is actually the correct solution.

### CODE REVIEW

File: webapp/src/Service/DOMJudgeService.php [hunk 1]
```diff
@@ -1228,13 +1228,15 @@ public function getAssetFiles(string $path): array
 
         $results = [];
         foreach (scandir($customDir) as $file) {
-            foreach (static::MIMETYPE_TO_EXTENSION as $extension) {
+            foreach (array_merge(['css'],static::MIMETYPE_TO_EXTENSION) as $extension) {
```
Comment (nickygerritsen) [inline webapp/src/Service/DOMJudgeService.php:1231]: We also need .js in that case

File: webapp/src/Service/DOMJudgeService.php [hunk 2]
```diff
@@ -1228,13 +1228,15 @@ public function getAssetFiles(string $path): array
...
                 if (strpos($file, '.' . $extension) !== false) {
                     $results[] = $file;
                 }
             }
         }
 
+        //dump($results);
```
Comment (nickygerritsen) [inline webapp/src/Service/DOMJudgeService.php:1238]: Dumpiedump

File: webapp/src/Service/DOMJudgeService.php (hunk 1 above)
Comment (nickygerritsen) [reply webapp/src/Service/DOMJudgeService.php:1231]: Also missing space after ,

Comment (vmcj) [reply webapp/src/Service/DOMJudgeService.php:1231]: But where did this stop working.... and why?

Comment (nickygerritsen) [reply webapp/src/Service/DOMJudgeService.php]: I think when I rewrote everything to not use the symfony DI container detection

**Pull request merged**